core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -606,6 +606,33 @@ public void resetState() {
     mPathProperties.resetState();
   }
 
+  @Override
+  public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMap) {
+    Map<String, Boolean> result = new HashMap<>();
+    int successCount = 0;
+    for (Map.Entry<String, String> entry : propertiesMap.entrySet()) {
+      try {
+        PropertyKey key = PropertyKey.fromString(entry.getKey());
+        if (key.isDynamic()) {
+          String oldValue = ServerConfiguration.get(key);
+          ServerConfiguration.set(key, entry.getValue(), Source.RUNTIME);
+          result.put(entry.getKey(), true);
+          successCount++;
+          LOG.debug("Update property {} from {} to {}",
+              key.getName(), oldValue, ServerConfiguration.get(key));
```

Comment:
Think this twice, I actually think we shall `LOG.info` this change as it may help debug.
Without knowing what configuration is changed during runtime, it can be hard to reason.

```suggestion
          LOG.info("Property {} has been updated to \"{}\" from \"{}\"",
              key.getName(), entry.getValue(), oldValue);
```

Reply:
Ok